docs/apache-airflow/upgrading-to-2.rst

```diff
@@ -325,6 +325,13 @@ the only supported UI.
 
 **Breaking Change in OAuth**
 
+.. note::
+
+    When multiple replicas of the airflow-web pods are running in Kubernetes they
+    need to share the same *secret_key* to access the same user session across pods. Inject
+    this via the environment and rotate it regularly like the fernet_key to ensure security.
+    The 1.10.14 bridge-release has this feature.
```

Comment:
This is too specific:

- Don't mention pods; this applies in or out of kubernetes usage.
- Specifying via environment is not required -- any config mechanism is valid.
- Please remove reference to rotating keys
- _All_ versions had this feature, it's just the previously they all had an insecure default so it wasn't noticed.